modules/core/src/main/scala/org/scalasteward/core/git/CommitMsg.scala

```diff
@@ -53,20 +53,41 @@ object CommitMsg {
         CommitMsg(title = title)
       },
       g => {
-        val artifactsWithVersions = g.updates
-          .groupBy(_.nextVersion)
-          .map { case (version, updates) =>
-            s"${updates.map(u => s"${u.artifactId.name}").mkString(", ")} to ${version.value}"
-          }
-          .mkString(" - ")
-
-        val defaultTitle =
-          s"Update for group ${g.name}${baseBranch.fold("")(branch => s" in ${branch.name}")}: $${artifactVersions}"
-
-        val title = g.title
-          .getOrElse(defaultTitle)
-          .replace("${artifactVersions}", artifactsWithVersions)
-        CommitMsg(title = title)
+        CommitMsg(title = groupMessage(group = g, baseBranch))
       }
     )
+
+  def groupMessage(group: Update.Grouped, baseBranch: Option[Branch]): String = {
+    def innerGroupMessage(shortNotation: Boolean) = {
+      val artifactsWithVersions = group.updates
```

Comment:
**question:** Could we add a test for this part?

<sub>This comment follows the [**conventionalcomments.org**](https://conventionalcomments.org) standard</sub>

Reply:
If you agree with an implementation like this: Of course :)

Reply:
oh yeah! I think is a good approach 😊

Reply:
I've added the tests.